oak_remote_attestation_verification/src/verifier.rs

```diff
@@ -159,58 +176,54 @@ pub fn verify_endorsement_statement(
     Ok(())
 }
 
-/// Verifies that the rekor public key in the given BinaryAttestation is either the same as the
+/// Verifies that the Rekor public key in the given attestation is either the same as the
 /// given public key, signed by it, or derived from it.
```

Comment:
Is this comment correct? It seems we are only checking equality

Reply:
Done.